### DISCUSSION TITLE
arm build Pi4

### DISCUSSION THREAD

**Discussion by JA16122000** [mrlt8/docker-wyze-bridge#729] [Q&A]
How can i test a "arm" build? [This](https://github.com/mrlt8/docker-wyze-bridge/issues/294#issuecomment-1043740792)
`        build:`
`            context: ./app`
`            dockerfile: Dockerfile.armv6`

context: ./app wont work

**Comment by mrlt8** [comment]
armv6 is not supported right now. 

However, build instructions for the Pi 4 can be found in the wiki:
https://github.com/mrlt8/docker-wyze-bridge/wiki/Raspberry-Pi-(armv7-and-arm64)#build-from-source

**Reply by JA16122000** [reply]
Tried that, but 
context: ./app wont work
is not recognized

**Reply by mrlt8** [reply]
That should work from the root directory of the project, but you may need to modify the script to match your local development environment.

**Reply by JA16122000** [reply]
Well, i don't know how to do that haha!
Are there any plans so far, that "arm" is ready to use anytime nearby?

**Reply by mrlt8** [reply]
amrv7 and arm64 are both already supported and included in the multi-arch builds.
